csrc/id_model/indexing.cpp

```diff
@@ -509,8 +514,31 @@ IndexingInfo TensorIndexer::computeIndex(
     index_compute.propagate(expr_group, direction);
   }
 
-  IndexingInfo info{traversal_path, index_compute.indexMap()};
+  IndexingInfo info{loop_domains, traversal_path, index_compute.indexMap()};
   return info;
 }
 
+std::unordered_map<Val*, Val*> TensorIndexer::getIndexReplacementMap(
```

Comment:
I think we should name this `getVectorizedIndexReplacementMap` since we skip non-vectorized groups in line 531.

Reply:
Let me leave it as is. I may add more stuff here.